X16 Reference - 05 - Math Library.md

```diff
@@ -5,85 +5,178 @@ The Commander X16 contains a floating point Math library with a precision of 40
 
 The full documentation of these functions can be found in the book [C128 Developers Package for Commodore 6502 Development](http://www.zimmers.net/anonftp/pub/cbm/schematics/computers/c128/servicemanuals/C128_Developers_Package_for_Commodore_6502_Development_(1987_Oct).pdf). The Math Library documentation starts in Chapter 13. (PDF page 257)
 
-The following functions are available from machine language code after setting the ROM bank to 4.
+The following functions are available from machine language code after setting the ROM bank to 4, which is the default.
 
 ## Format Conversions
 
-| C128  | C65   | X16   | Symbol   | Description                            |
-|-------|-------|-------|----------|----------------------------------------|
-| \$AF00 | \$7F00 | \$FE00 | `AYINT`  | convert floating point to integer      |
-| \$AF03 | \$7F03 | \$FE03 | `GIVAYF` | convert integer to floating point      |
-| \$AF06 | \$7F06 | \$FE06 | `FOUT`   | convert floating point to ASCII string |
-| \$AF09 | \$7F09 | \$FE09 | `VAL_1`  | convert ASCII string to floating point<br/>*[not yet implemented]*|
-| \$AF0C | \$7F0C | \$FE0C | `GETADR` | convert floating point to an address   |
-| \$AF0F | \$7F0F | \$FE0F | `FLOATC` | convert address to floating point      |
+| Address | Symbol   | Description                                                        |
+|---------|----------|--------------------------------------------------------------------|
```

Comment:
The content looks good, but the `$` symbols need to be escaped unless they're either inside an HTML table or inside a fenced code block. GitHub shows them correctly anyway, but the PDF creation has a more strict syntax.

Reply:
I've added the escaping back in. There's still an issue with the attempt to get the TeX formatted formula into the PDF